src/AddressRegistry.Api.Oslo/Address/AddressController.cs

```diff
@@ -90,6 +91,29 @@ public async Task<IActionResult> List(CancellationToken cancellationToken = defa
             return Ok(result);
         }
 
+        /// <summary>
+        /// Zoek straatnamen / adressen.
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <response code="200">Als de opvraging van de zoekopdracht gelukt is.</response>
+        /// <response code="500">Als er een interne fout is opgetreden.</response>
+        [HttpGet("zoeken")]
+        [Produces(AcceptTypes.JsonLd)]
+        //[ProducesResponseType(typeof(TotaalAantalResponse), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
+        //[SwaggerResponseExample(StatusCodes.Status200OK, typeof(TotalCountOsloResponseExample))]
```

Comment:
The commented attributes can go away or should be fixed?

Reply:
search is for later, PR has pivoted away to replace list